### ISSUE TITLE
Looking forward to release soon!

### ISSUE THREAD

**Issue by ykq527** [iVMCL/PaCaViT#1]
Fantastic work!  Does the team have a plan for when the code will be released?

**Comment by ykq527** [comment]
Hello, I have some questions about your paper that I would like to inquire about. In Formula 5, "softmax along N," when the feature map resolution is relatively large, N will be large as well. After the softmax, each element will become very small and relatively close to each other. I am wondering if there would be any issues with this approach? For example, when the input feature map resolution is (16, 12), N=192, the result after the softmax may fluctuate between 0.0050 and 0.0053.

**Comment by iVMCL** [comment]
Dear ykq527, Thank you for your interest. We have release the code, and in the process of uploading the pretrained checkpoints.

**Comment by iVMCL** [comment]
Please let us know if you have any questions of the code. We close this issue.
